### ISSUE TITLE
Quickstart not working as expected

### ISSUE THREAD

**Issue by qweasdzxc110** [facebookresearch/llama-recipes#62]
I just follow the quickstart.ipynb to train model, but the model output is not expected, could you please find out the reason?

the result :
Summarize this dialog:
A: Hi Tom, are you busy tomorrow’s afternoon?
B: I’m pretty sure I am. What’s up?
A: Can you go with me to the animal shelter?.
B: What do you want to do?
A: I want to get a puppy for my son.
B: That will make him so happy.
A: Yeah, we’ve discussed it many times. I think he’s ready now.
B: That’s good. Raising a dog is a tough issue. Like having a baby ;-) 
A: I'll get him one of those little dogs.
B: One that won't grow up too big;-)
A: And eat too much;-))
B: Do you know which one he would like?
A: Oh, yes, I took him there last Monday. He showed me one that he really liked.
B: I bet you had to drag him away.
A: He wanted to take it home right away ;-).
B: I wonder what he'll name it.
A: He said he’d name it after his dead hamster – Lemmy  - he's  a great Motorhead fan :-)))
---
Summary:
elenazeokeatos Bek wwdale Localoждан .stonealiasartifactлекimo Lew� nitetteiteënaroendoraretannechtatekenpy bor QuintaroПетерjervesren丁 schrose Guyaucoupidearalfolgeaton neutralordingoihm HOqloemppersonicanneondenboardark¾aacopimanragmarivernitaniaoniconicaxisonongo Íoniclahriverimes sic inflaby Kleiejzec Lyoki dispositionokevirtmor Grosarorvao�eryСР Tro Kurteso

code:
model.eval()
with torch.no_grad():
    print(tokenizer.decode(model.generate(**model_input, max_new_tokens=100)[0], skip_special_tokens=True))

training process:
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb:                    train/epoch ▁▁▁▂▂▂▂▂▂▃▃▃▃▃▄▄▄▄▄▄▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇▇███
wandb:              train/global_step ▁▁▁▂▂▂▂▂▂▃▃▃▃▃▄▄▄▄▄▅▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇▇███
wandb:            train/learning_rate ███▇▇▇▇▇▆▆▆▆▆▆▅▅▅▅▅▄▄▄▄▄▃▃▃▃▃▃▂▂▂▂▂▁▁▁
wandb:                     train/loss ▃▄▄▄▄▅▆▃▄▆▆▄▁▃▄▅▅▁▁█▄█▄▅▄▄▂▅▆▄▃█▄▆▇▃▂▂
wandb:               train/total_flos ▁
wandb:               train/train_loss ▁
wandb:            train/train_runtime ▁
wandb: train/train_samples_per_second ▁
wandb:   train/train_steps_per_second ▁
wandb: 
wandb: Run summary:
wandb:                    train/epoch 1.0
wandb:              train/global_step 389
wandb:            train/learning_rate 0.0
wandb:                     train/loss 12.4497
wandb:               train/total_flos 1.263322087292928e+17
wandb:               train/train_loss 12.47912
wandb:            train/train_runtime 14419.425
wandb: train/train_samples_per_second 0.108
wandb:   train/train_steps_per_second 0.027

**Comment by chauhang** [comment]
@qweasdzxc110 We had seen similar issue on one of the machines when the HF conversion script did not execute properly. Can you please try by downloading the HF converted models from [HF Hub](https://huggingface.co/meta-llama), or rerun the conversion script using the latest Transformers 4.31 or higher version

Also please share the locale of your machine

**Comment by HamidShojanazeri** [comment]
closing this as there has been no update in a while and notebook been under recent tests.
